python/tidy/tidy.py

```diff
@@ -148,12 +148,19 @@ def progress_wrapper(iterator):
 
 
 def git_changes_since_last_merge(path):
-    args = ["git", "log", "-n1", "--committer", "noreply@github.com", "--format=%H"]
+    base_branch = "upstream/main"
+    if os.getenv("CI"):
+        base_branch = "origin/main"
+    else:
+        remotes = subprocess.check_output(["git", "remote"], universal_newlines=True).splitlines()
+        if "upstream" not in remotes:
+            subprocess.check_call(["git", "remote", "add", "upstream", "https://github.com/servo/servo.git"])
```

Comment:
While someone could set wrong upstream remote purposefully (but I think it's safe to ignore this if CI will catch this), I actually love how this will give everybody upstream remote as a side effect.